docker-compose.yml

```diff
@@ -74,6 +74,7 @@ x-sentry-defaults: &sentry_defaults
     SENTRY_EVENT_RETENTION_DAYS:
     SENTRY_MAIL_HOST:
     SENTRY_MAX_EXTERNAL_SOURCEMAP_SIZE:
+    # Set this value if you plan on using the Suggested Fix Feature
```

Comment:
What is this feature? Are there docs for it anywhere? This implies we're using ChatGPT under the hood for it?

Reply:
Docs are here: 
https://docs.sentry.io/product/issues/issue-details/ai-suggested-solution/

We're using ChatGPT to recommend solutions for issues